package.json

```diff
@@ -461,6 +461,21 @@
 					"default": "auto",
 					"markdownDescription": "%githubPullRequests.createDefaultBaseBranch.description%"
 				},
+				"githubPullRequests.experimental.chat": {
+					"type": "boolean",
+					"markdownDescription": "%githubPullRequests.experimental.chat.description%",
+					"default": false
+				},
+				"githubPullRequests.experimental.notificationsView": {
+					"type": "boolean",
+					"markdownDescription": "%githubPullRequests.experimental.notificationsView.description%",
+					"default": false
+				},
+				"githubPullRequests.experimental.useQuickChat": {
+					"type": "boolean",
+					"markdownDescription": "%githubPullRequests.experimental.useQuickChat.description%",
+					"default": true
```

Comment:
Let's flip this one to false since the Chat panel is moving to the auxiliary sidebar

Reply:
There are also some rendering issues when running the commands in the quick chat. The response is streamed but it disappears sometimes from the quick chat. Does not seem to happen in panel chat.